README.md

```diff
@@ -12,5 +12,6 @@ documentation and the `void-docs.7` man page. It requires the following
 dependencies:
 
 - `mdBook`
+- `mdbook-linkcheck`
```

Comment:
`mdbook-linkcheck` is a dependency for contribution, which is mentioned here https://docs.voidlinux.org/contributing/void-docs/style-guide.html#checking-links (now that I think of it, it should be mentioned more clearly elsewhere - perhaps in the CONTRIBUTING file).

The dependencies in the README, on the other hand, are only for building the HTML and roff (man pages) versions of this same Handbook. I am using them for building the void-docs package (still pending, for now).